### DISCUSSION TITLE
Translation strings via API

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#15782] [Feature Requests]
Hello

I guess currently currently translation strings are not directly accessible via API (for example GET /translations) - that would be very useful for the case where web app designed to display content from Directus has multilingual functionality.

Hembo

**Comment by HemboPagi** [comment]
Ignore it, I found that they are accessible via settings

**Reply by azrikahar** [reply]
Just noticed it's missing from the docs, thanks for pointing it out! Included PR: directus/docs#156
